python/arcticdb/version_store/_normalization.py

```diff
@@ -769,9 +772,22 @@ def normalize(self, item, string_max_len=None, dynamic_strings=False, coerce_col
             norm_meta.df.common.columns.name = item.columns.name
         else:
             norm_meta.df.common.columns.fake_name = True
+
+        sort_status = _SortedValue.UNKNOWN
+        if hasattr(item.index, "is_monotonic_increasing"):
```

Comment:
Should really `MultiIndex`

Reply:
This is a great point actually - if multiindex is set is `index` not set?

Reply:
`MultiIndex` is just a sub-class of `Index`, but it doesn't directly have `is_monotonic_increasing` as it wouldn't make sense. Will have to drill down to the first level of a `MultiIndex`.

Reply:
better to discuss with Matt

Reply:
The logic we have right now already works for multi index. The reason is because _index_to_records function is doing a reset index for the index levels beyond than zero. which basically, converts the index to its single version (relying only in the first level). I added a test for it